README.md

```diff
@@ -37,7 +37,7 @@ In the new frontier of DevOps it is becoming more and more popular to interact w
 
 ## Quick start
 
-```
+```python
```

Comment:
This is not python code but shell!

Reply:
That's what intrigued me too!

Reply:
Ha I totally missed that. That'll teach me for doing such a speedy review!

Reply:
Fixed on master!

Reply:
@jacobtomlison Be careful of documentation change PR's. Most of the times these are just to gain some swag or stickers and not really contribute to the true documentation.

I always stay away from PR's that change 4 lines  in documentation!

Reply:
Totally agree and I nearly rejected it on those grounds, however at the moment I am trying to spread the word about opsdroid. So I'm happy to send out the stickers. If it starts happening more often I will become more strict.